playbooks/roles/edxapp/files/update_theme.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT license. See LICENSE file on the project webpage for details.
+
+set -x
+
+sudo cp /oxa/oxa-tools/config/server-vars.yml /edx/app/edx_ansible/
```

Comment:
Not sure that I agree with overwriting /edx/app/edx_ansible/server-vars.yml.  If we do this, it would probably be better to move this code into update.sh and make a backup of the file prior to overwriting.